chef_master/source/custom_resources.rst

```diff
@@ -857,7 +980,7 @@ For example, the following properties define ``username`` and ``password`` prope
 .. end_tag
 
 Block Arguments
-+++++++++++++++++++++++++++++++++++++++++++++++++++++
+-----------------------------------------------------
```

Comment:
Why a different header size here ? This would break the overall presentation, or did I miss something ?